package/MDAnalysis/analysis/psa.py

```diff
@@ -575,7 +575,7 @@ def dist_mat_to_vec(N, i, j):
       int, index (of the matrix element) in the corresponding distance vector
     """
 
-    if not (isinstance(N, int) or isinstance(i, int) or isinstance(j, int)):
+    if not (isinstance(N, numbers.Integral) or isinstance(i, numbers.Integral) or isinstance(j, numbers.Integral)):
```

Comment:
What @kain88-de means is that you should add a few lines in `test_psa.py` with numpy integer as inputs. The function should work of we give `N=numpy.int32(4)` or `N=numpy.int16(4)`. and this has to be tested. It is likely that the input values for `dist_mat_to_vec` will come from a numpy array at some point, then their type will not be a plain `int`.